test/test_api.cpp

```diff
@@ -12,12 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <gtest/gtest.h>
```

Comment:
I will change it to its correct place, but ament_cpplint currently considers it a C header, which is why I reordered it in the first place.